src/index.test.ts

```diff
@@ -1,10 +1,14 @@
 import { deepEqual } from "node:assert/strict";
 import { test } from "node:test";
 
-function greet() {}
+const greet = (name: string) => {
+  return {
+    message: `Welcome, ${name}!`
+  };
+};
 
-test("should greet person", () => {
-  const result = greet("Alek");
+test('should greet person', () => {
```

Comment:
Please configure your editor to use Prettier for formatting to avoid reformatting already formatted code.
Really nice to see you using a formatter by the way! Good job!